utils/webassembly/build-toolchain.sh

```diff
@@ -61,7 +61,8 @@ cd $TMP_DIR/$TOOLCHAIN_NAME
 
 # Merge wasi-sdk and toolchain
 cp -r $WASI_SDK_PATH/lib/clang usr/lib
-cp $WASI_SDK_PATH/bin/* usr/bin
+cp -a $SOURCE_PATH/build/Ninja-ReleaseAssert/llvm-*/bin/clang* usr/bin
```

Comment:
Please use `--llvm-install-components` instead of depending on build directory path :pray:
https://gist.github.com/kateinoigakukun/af8226e41a6db4d12706d8faa187cf3b